packages/core/src/components/tabs/tabs.tsx

```diff
@@ -126,6 +126,7 @@ export interface TabsState {
  * @see https://blueprintjs.com/docs/#core/components/tabs
  */
 export class Tabs extends AbstractPureComponent<TabsProps, TabsState> {
+    /** @deprecated Use `Tab` instead */
```

Comment:
This seems a bit vague/confusing by itself since the thing we are deprecating is also called "Tab". Maybe we should reference the Tab **_component_** here (or link to it?)

Reply:
Good point, updated the comment, let me know what you think.